apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java

```diff
@@ -62,7 +62,7 @@ public static <T> T load(String className,
             try {
                 pluginLoader = EXTEND_PLUGIN_CLASSLOADERS.get(targetClassLoader);
                 if (pluginLoader == null) {
-                    pluginLoader = new AgentClassLoader(targetClassLoader);
+                    pluginLoader = AgentClassLoader.getClassLoader(targetClassLoader);
```

Comment:
So, you use the cached plugin classloader? I am not following.

EXTEND_PLUGIN_CLASSLOADERS is already a cache, why AgentClassLoader builds a cache again?

Reply:
`EXTEND_PLUGIN_CLASSLOADERS` is a storage place to get the `AgentClassLoader`, and the `AgentClassLoader.getClassLoader` method is to get all the classes in the bundle and the class loader of the skywalking plugin, which can be used for other places later